src/components/chat/chat-thread.scss

```diff
@@ -37,34 +37,55 @@
       background-color: $charcoal-light-8;
       color: $charcoal;
       border: solid 1px $charcoal-light-2;
-      &:hover {
-        opacity: .5;
-      }
-      &.selected {
-        background-color: $comment-select-green-dark;
-        border: none;
-      }
       &.problems {
         background-color: $problem-orange-light-7;
+        &:hover{
+          background-color: $problem-comment-hover;
+        }
+        &.selected {
+          border: none;
+          background-color: $comment-select-yellow;
+        }
       }
       &.teacher-guide {
         background-color: $learninglog-green-light-7;
+        &:hover{
+          background-color: $teacherguide-comment-hover;
+        }
+        &.selected {
+          border: none;
+          background-color: $comment-select-yellow;
+        }
       }
       &.my-work {
         background-color: $workspace-teal-light-6;
+        &:hover{
+          background-color: $mywork-comment-hover;
+        }
+        &.selected {
+          border: none;
+          background-color: $comment-select-yellow;
+        }
       }
       &.class-work {
         background-color: $classwork-purple-light-7;
+        &:hover{
+          background-color: $classwork-comment-hover;
+        }
+        &.selected {
+          border: none;
+          background-color: $comment-select-yellow;
+        }
```

Comment:
Since the `.selected` styling is the same for all document types, could we define it outside of the individual documents so it's not repeated four times? Or does that cause some other problems?

Reply:
I believe I was running into problems earlier (hence the 4 times)  but now I got it working, so I just pushed it up with one outer class. 


```
 &.selected {
        border: none;
        background-color: $comment-select-yellow;
      }
```

Reply:
I'm going to go ahead and merge this then.